app/forms/access-util.tsx

```diff
@@ -14,20 +17,39 @@ import {
 } from '@oxide/api'
 import { Badge } from '@oxide/design-system/ui'
 
+import { RadioFieldDyn } from '~/components/form/fields/RadioField'
+import { HL } from '~/components/HL'
 import { type ListboxItem } from '~/ui/lib/Listbox'
+import { Message } from '~/ui/lib/Message'
+import { Radio } from '~/ui/lib/Radio'
 import { capitalize } from '~/util/str'
 
 type AddUserValues = {
   identityId: string
-  roleName: RoleKey | ''
+  roleName: RoleKey
```

Comment:
This `''` value was me a long time ago, but I realized it's silly and we should just make viewer the default. This also required reversing the order so viewer is first, otherwise the default value is at the bottom, which is weird.